test/modules/packages/mpi/multilocale/hello-chapel-nl-1/hello.chpl

```diff
@@ -7,6 +7,17 @@ coforall loc in Locales do on loc {
   for irank in 0.. #size {
     if irank == rank then
       writef("Hello, Chapel! This is MPI rank=%i of size=%i, on locale.id=%i\n",rank, size, here.id);
+
+    /* Why the double barrier?
+
+       A single barrier is to ensure that the code outputs in the correct sequence, and
+       doesn't overwrite. This could be achieved with a single Barrier.
+
+       Having a direct call into the MPI_Barrier also tests that the first Barrier prevents
+       deadlocks when running with CHPL_TASKS=qthreads. This second MPI_Barrier call stands in for any blocking
```

Comment:
Inconsistent line width in these comments looks a little weird, but no big deal if you don't feel like fixing it.